packages/seacas/libraries/ioss/src/Ioss_DatabaseIO.C

```diff
@@ -877,6 +877,16 @@ namespace Ioss {
   void DatabaseIO::set_block_omissions(const std::vector<std::string> &omissions,
                                        const std::vector<std::string> &inclusions)
   {
+    if (!omissions.empty() && !inclusions.empty()) {
```

Comment:
I don't think this is the same as what the asserts were checking.  The assert was assembly incluion and block omission or assembly omission and block inclusion.

This here is block omission and inclusion...